exif.h

```diff
@@ -136,6 +136,7 @@ class EXIFInfo {
     double FocalLengthMax;          // Max focal length (mm)
     double FocalPlaneXResolution;   // Focal plane X-resolution
     double FocalPlaneYResolution;   // Focal plane Y-resolution
+    unsigned FocalPlaneResolutionUnit; // Focal plane resolution unit
```

Comment:
Default initialization in the cpp is missing
`LensInfo.FocalPlaneResolutionUnit= 0;`